src/styles/utilities/_responsive-embed.scss

```diff
@@ -0,0 +1,15 @@
+// the `video` modifier assumes an aspect ratio of 16:9 (height 9 / width 16 = 0.5625)
+.responsive-embed__container--video {
```

Comment:
I guess we could drop `_container` here and just use `.responsive-embed` as the container?

Also, it feels a little weird to have a modifier without an actual base style. Plus with BEM, an element should have both the base class and the modifier class. Maybe have a base with `padding-top: 100%;' and just change that with the modifier?

So altogether:
```
.responsive_embed {
  position: relative;
  overflow: hidden;
  padding-top: 100%;

  &--video {
    padding-top: 56.25%;
  }
}
```

I'm not massively opinionated on what I've just suggested, so happy to hear any reasons to the contrary.

In terms of the approach, it's basically *the* way of doing responsive iframe embeds.

Reply:
Thanks - that makes a lot more sense, will update!

Reply:
My only other issue was that it's a utility class without a `.u-` prefix (and also somewhat more complex than a standard utility one-liner), but it doesn't fit nicely within the context of `tools`, `settings`, `objects` etc at all.